# Reorder include directories to allow build with multiple ffmpeg installed.
# This patch must NOT be used, if FFMPEG_INCLUDE_DIRS == ${LOCALBASE}/include
#
--- ./CMakeLists.txt.orig	2013-03-01 07:07:31.562142000 +0000
+++ ./CMakeLists.txt	2013-03-06 00:33:28.482362201 +0000
@@ -7,11 +7,11 @@
 find_package(FFmpeg COMPONENTS AVCODEC AVFORMAT SWSCALE)
 
 include_directories(
-    ${KDE4_INCLUDES}
-    ${CMAKE_CURRENT_BINARY_DIR}
+    ${FFMPEG_INCLUDE_DIR}
     ${PC_AVCODEC_INCLUDEDIR}
     ${PC_AVFORMAT_INCLUDEDIR}
-    ${FFMPEG_INCLUDE_DIR}
+    ${KDE4_INCLUDES}
+    ${CMAKE_CURRENT_BINARY_DIR}
     )
 
 # Certain versions of FFMPEG need this to be defined
